librz/core/rtr_http.c

```diff
@@ -132,16 +132,12 @@ static int rz_core_rtr_http_run(RzCore *core, int launch, int browse, const char
 	core->config = newcfg;
 
 	rz_config_set(core->config, "asm.cmt.right", "false");
-#if 0
-	// WHY
-	rz_config_set (core->config, "scr.html", "true");
-#endif
 	rz_config_set_i(core->config, "scr.color", COLOR_MODE_DISABLED);
 	rz_config_set(core->config, "asm.bytes", "false");
 	rz_config_set(core->config, "scr.interactive", "false");
-	eprintf("Starting http server...\n");
-	eprintf("open http://%s:%d/\n", host, atoi(port));
-	eprintf("rizin -C http://%s:%d/cmd/\n", host, atoi(port));
+	RZ_LOG_WARN("core: Starting http server...\n");
```

Comment:
RZ_LOG_INFO in this case? This does not seem a warning or a problematic thing, just an info.